src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -293,12 +293,19 @@ trait EventFormats {
     )
   }
   implicit lazy val LightDeploymentFailedWrites: Writes[DeploymentFailed] = Writes { event =>
-    Json.obj(
+    val serializedEvent = Json.obj(
       "id" -> event.id,
       "plan" -> LightDeploymentPlanWrites.writes(event.plan),
       "eventType" -> "deployment_failed",
       "timestamp" -> Timestamp.now().toString
     )
+
+    event.reason match {
+      case Some(reason) =>
+        serializedEvent ++ Json.obj("reason" -> reason)
```

Comment:
Please document the `reason` field in the RAML docs

Reply:
I don't think we have the event types in RAML anywhere. Do we?